apps/web/src/components/Layout/MaintenanceBanner.tsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable import/no-duplicates */
+import React from "react";
+import { useIntl } from "react-intl";
+import isAfter from "date-fns/isAfter";
+import isBefore from "date-fns/isBefore";
+import isEqual from "date-fns/isEqual";
+
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { getRuntimeVariable } from "@gc-digital-talent/env";
+import { Alert } from "@gc-digital-talent/ui";
+
+const MaintenanceBanner = () => {
+  const intl = useIntl();
+  // Outage time range, date, starttime
+  const maintenanceBannerPublicDate = getRuntimeVariable(
+    "MAINTENANCE_BANNER_PUBLIC_DATE",
+  );
+  const serverMaintenanceDate = getRuntimeVariable(
+    "MAINTENANCE_BANNER_START_DATE",
+  );
+  const serverMaintenanceDuration = getRuntimeVariable(
+    "MAINTENANCE_BANNER_DURATION",
+  );
+
+  const startDate = serverMaintenanceDate
+    ? formatDate({
+        date: parseDateTimeUtc(serverMaintenanceDate),
+        formatString: `PPPPpp`,
+        intl,
+      })
+    : null;
+
+  const duration =
+    serverMaintenanceDuration && !!Math.abs(+serverMaintenanceDuration)
+      ? Math.abs(+serverMaintenanceDuration)
+      : null;
+
+  const showMaintenanceBanner =
+    maintenanceBannerPublicDate && serverMaintenanceDate
+      ? (isEqual(parseDateTimeUtc(maintenanceBannerPublicDate), new Date()) ||
+          isAfter(parseDateTimeUtc(maintenanceBannerPublicDate), new Date())) &&
+        isBefore(
```

Comment:
Did you mean to compare the two environment variables here?

Reply:
I think they just needed to be reversed right. 

If current date is equal or after the public date time then show the banner.